dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcTeiEnrollmentsAnalyticsTableManager.java

```diff
@@ -238,7 +238,7 @@ protected void populateTable(
     }
 
     removeLastComma(sql)
-        .append(" from programinstance pi")
+        .append(" from enrollment pi")
```

Comment:
When do you plan to rename aliases? We should probably settle on new ones, maybe `en` for `enrollment`, `ev` for `event` and `te` for `trackedentity` 🤔 

@enricocolasante

Reply:
@teleivo @enricocolasante 
https://dhis2.atlassian.net/browse/TECH-1596 https://dhis2.atlassian.net/browse/TECH-1597